spaces/S000145/properties/P000086.md

```diff
@@ -0,0 +1,8 @@
+---
+space: S000145
+property: P000086
+value: true
+---
+
+Let $p, q \in X$ be arbitrary distinct points.
+Define $f(x) = \begin{cases} q & x = p, \\ p & x = q, \\ x & \text{otherwise}, \end{cases}$ then $f(x)$ is involutary and continuous since for every open subset $U$ and every point $x \in X$, both $U \setminus \{x\} = U \cap \left( X \setminus \{x\} \right)$ and $U \cup \{x\}$ are open.
```

Comment:
```suggestion
Define $f(x) = \begin{cases} q & \text{if }x = p, \\ p & \text{if }x = q, \\ x & \text{otherwise}. \end{cases}$ Then $f(x)$ is involutary and continuous since for every open subset $U$ and every point $x \in X$, both $U \setminus \{x\} = U \cap \left( X \setminus \{x\} \right)$ and $U \cup \{x\}$ are open.
```